azure-pipelines.yml

```diff
@@ -0,0 +1,25 @@
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#define-a-repositories-resource
+resources:
+  repositories:
+    - repository: IOMDevopsRepo
+      type: git
+      name: order-iom-partner-devops
+      ref: task/76533-ci-pipeline
```

Comment:
this fallback is not for prod, right ? and will be deleted after ...

Reply:
All projects have use the main branch of the pipeline template. But currently there is no pipeline template in main. This has to be changed later, when template is merged to its final position.